### ISSUE TITLE
Configure Path Monitoring for a Static Route

### ISSUE THREAD

**Issue by antoinekh** [PaloAltoNetworks/pan-os-ansible#542]
### Is your feature request related to a problem?

It's a need to have the same feature in Ansible than in the GUI

### Describe the solution you'd like

Option to configure Configure Path Monitoring for a Static Route
https://docs.paloaltonetworks.com/pan-os/10-1/pan-os-networking-admin/static-routes/configure-path-monitoring-for-a-static-route

### Describe alternatives you've considered

I need to try it with the restapi

### Additional context
A screenshot of the parameters I want to be able to customise with Ansible
![image](https://github.com/PaloAltoNetworks/pan-os-ansible/assets/52744345/e3c3e172-8208-4a05-9f63-d381f702e230)

**Comment by bmeyer144** [comment]
bump. You can enable path monitoring and select the failure condition but the static route module does not support adding a "path monitoring parameter"

<img width="356" alt="path-monitor-png" src="https://github.com/PaloAltoNetworks/pan-os-ansible/assets/118846170/62082ec1-c8e1-47bb-bc6d-9564baebad30">

**Comment by antoinekh** [comment]
> bump. You can enable path monitoring and select the failure condition but the static route module does not support adding a "path monitoring parameter." These can only be added with the XML-API or the GUI
> 
> <img alt="path-monitor-png" width="356" src="https://private-user-images.githubusercontent.com/118846170/305533679-62082ec1-c8e1-47bb-bc6d-9564baebad30.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDgxMTEzMzksIm5iZiI6MTcwODExMTAzOSwicGF0aCI6Ii8xMTg4NDYxNzAvMzA1NTMzNjc5LTYyMDgyZWMxLWM4ZTEtNDdiYi1iYzZkLTk1NjRiYWViYWQzMC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwMjE2JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDIxNlQxOTE3MTlaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yYjgzOTM5OGI2NmMyZjQwMzVhYzhhODg1NmE4NzhiNmQ0MDg3YTZjZmJjZDE4YWM2OTllZDNhMDFhZjUwNTVlJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.6UzJn61rIXLtqWgCCBNscEGku6qiiHn-DgxVm9N79_k">

Yes I use the GUI for now, and I will use CLI with op ansible module or the API to automate it. but that’s bad that there is again a missing feature in the Ansible collection
